.travis.yml

```diff
@@ -12,9 +13,10 @@ install:
     - "pip install -r test_requirements.txt"
     - "pip install -r requirements.txt"
     - "pip install -r doc_requirements.txt"
-    - "sudo ./scripts/install_browsermob_proxy.sh"
+    - "./scripts/install_browsermob_proxy.sh"
 script:
     - "pylint bok_choy"
+    - "export PATH=browsermobtemp/browsermob-proxy/bin:$PATH"
```

Comment:
For travis, the location is one and the same. Saying it with variables (that are not exposed to travis, so we hardcode them), the path to the bin dir is `$TEMP_BROWSERMOB_DIR/browsermob-proxy/bin`, which is the same as `$BROWSERMOB_INSTALL_LOC/bin`

For the travis script, we use the path relative to project root.